### PR TITLE
Creating pull request

### PR DESCRIPTION
**Pull request by cat-west**
This is the base HTML for my Adopt A Dog website.

### CODE REVIEW

File: checkout.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  
+                <label>Street Address
+                    <input type="text" name="contactLocation">
+                </label>
+
+                <label>City
+                        <input type="text" name="contactLocation">
+                    </label>
+
+                <label>State
```
Comment (ClovisI) [inline checkout.html]: Can you make this City, State, and Zip[lines 43 -100] display as one single line on your checkout.html?

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
```
Comment (jimmynono) [inline blog.html]: You'll want to delete these two empty lines

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
```
Comment (jimmynono) [inline blog.html:5]: Don't forget the meta tag from week on one all of your html pages

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+      <header>
+        <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <h1>Seattle's premier dog adoption agency</h1>
+        <img class="cart-img" src="images/cart-icon.svg" alt="Cart icon">
+        <a href="checkout.html" class="checkout-button">Checkout</a>
+      </header>
+
+      <nav>
+        <a href="index.html">Home&nbsp;&nbsp;/</a>
```
Comment (jimmynono) [inline blog.html]: Handling the / this way certainly works, but if you have time see if you can research how to use the after pseudo element to inject these with css. You'll want to use the content property. Totally not required, but would be a nice touch.

File: dogs.html [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+      <nav>
+        <a href="index.html">Home&nbsp;&nbsp;/</a>
+        <a href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a>
+        <a href="blog.html">&nbsp;&nbsp;Blog</a>
+      </nav>
+
+        <h2>Here are all of our dogs!</h2>
+
+        <div class="row">
```
Comment (jimmynono) [inline dogs.html]: I get what you're going to do with these classes of row and column. Maybe there are better more semantic words you could use?

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+              <div class="card">
+                  <img src="images/murphy-card.jpg" alt="Picture of Murphy">
+                  <h3>Murphy</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="checkout.html" class="card-button">Adopt</a>
+                </div>
+            </div>
+        
+                <div class="column">
```
Comment (jimmynono) [inline dogs.html]: Whoops...indentation is a little off

File: dogs.html [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+                <div class="card">
+                  <img src="images/duffy-card.jpg" alt="Picture of Duffy">
+                  <h3>Duffy</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+        
+            <div class="column">
```
Comment (jimmynono) [inline dogs.html]: These column classes might not be necessary

File: dogs.html [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+                </div>
+            </div>
+    
+            <div class="column">
+                <div class="card">
+                <img src="images/maximus-card.jpg" alt="Picture of Maximus">
+                <h3>Maximus</h3>
+                <h4>Cost to adopt: $123.45</h4>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <a href="#" class="card-button">Adopt</a>
```
Comment (jimmynono) [inline dogs.html]: Tip. These will be buttons that won't navigate anywhere, but will be used for simple javascript.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
```
Comment (jimmynono) [inline styles.css:4]: Whoops! Watch your indentation for this whole file. It's a a little wonky across the board

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,391 @@
+*{
```
Comment (jimmynono) [inline styles.css]: space after the *

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,391 @@
...
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
```
Comment (jimmynono) [inline styles.css]: make sure you have spaces after the colon. Remember, the formatting should look like...

h1 {
  color: #f2f2f2;
}

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,391 @@
...
+	background-color:#093C60;
+	color:white;
```
Comment (jimmynono) [inline styles.css]: Try not to mix and match how you're referencing colors. If you're using white, you can use #FFFFFF

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,391 @@
...
+    padding: 5% 13%;
+    max-height: 400px;
+    position: relative;
+}
+
+header h1 {
+	font-size: 1.1em;
+	font-family: 'Courgette', cursive;
+	margin-top: -10px;
```
Comment (jimmynono) [inline styles.css]: I'm leery about all these negative margins. Maybe dial down the padding on the header element? I'd rather have positive margin on the image. Negative margin can be tricky

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,391 @@
...
+    padding-left: 30px;
+}
+
+.cart-img {
+    display: inline;
+    max-width: 30px;
+    height: auto;
+    position: absolute;
+    top: 40px;
+    right: 100px;
```
Comment (jimmynono) [inline styles.css]: See if you can turn this into a percent and have it look decent.

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,391 @@
...
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
+	background-color:#093C60;
+	color:white;
+    padding: 5% 13%;
```
Comment (jimmynono) [inline styles.css]: This padding is really agressive and seems like it's causing layout issues a bit. Maybe dial it down a tad?

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,391 @@
...
+	font-size: 1.1em;
+	font-family: 'Courgette', cursive;
+	margin-top: -10px;
+    margin-left: -65px;
+    padding-left: 30px;
+    color: #75B441;
+}
+
+.header-logo {
+    max-width: 40%;
```
Comment (jimmynono) [inline styles.css]: Yay! Percents!

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,391 @@
...
+    background-color: #81B741;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 700;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 13px;
```
Comment (jimmynono) [inline styles.css]: Rems or Em's for fonts please. You can do the math 13/16 to get your REM value

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,391 @@
...
+    display: block !important;
+    position: absolute;
+    top: 70px;
+    right: 100px;
+}
+
+nav {
+	background-color: #ECECEC;
+    max-height: 5rem;
+    padding-left: 30px;
```
Comment (jimmynono) [inline styles.css]: You might want to consider a percent based padding

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,391 @@
...
+    overflow: hidden;
+    margin-bottom: 25px;
+}
+
+.main-content img {
+    float: left;
+    padding: 0 35px 20px 0;
+}
+
+section {
```
Comment (jimmynono) [inline styles.css]: The combo of margin right and padding left is a little wonky. Maybe set a max width on the section and margin: 0 auto, which should center it.

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,391 @@
...
+	margin-right:7%;
+	margin-bottom: 7%;
+	padding-left: 13%;
+}
+
+h2 {
+    font-family: 'Arbutus Slab', serif;
+    font-size: 2rem;
```
Comment (jimmynono) [inline styles.css]: REMS! Yay!

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,391 @@
...
+        text-decoration: none;
+    }
+    
+    footer p {
+        font-weight: 400;
+        font-size: 16px;
+    }
+
+
+.column {
```
Comment (jimmynono) [inline styles.css]: These might not be best served by floating. I'd prefer flexbox or CSS grid to lay these out

File: styles.css [hunk 22]
```diff
@@ -0,0 +1,391 @@
...
+    float: left;
+    width: 25%;
+    padding: 15px 10px;
+  }
+  
+  .row {
+    margin: 0 -5px;
+}
+  
+  .row:after {
```
Comment (jimmynono) [inline styles.css]: Yeah, I don't like this. If you get rid of the float strategy you can probably get rid of this.

File: styles.css [hunk 23]
```diff
@@ -0,0 +1,391 @@
...
+    display: table;
+    clear: both;
+  }
+
+  .card {
+    padding: 0px;
+    text-align: left;
+    border: #EDEDED;
+    border-style: solid;
+    border-width: 2px;
```
Comment (jimmynono) [inline styles.css]: You can shorthand border to 

border: 2px solid #EDEDED;

File: styles.css [hunk 24]
```diff
@@ -0,0 +1,391 @@
...
+  }
+
+  div h3 {
+    font-family: 'Arbutus Slab', serif;
+    color: #093C60;
+    font-size: 1.2rem;
+    margin-left: 10px;
+    margin-bottom: -5px;
```
Comment (jimmynono) [inline styles.css]: Yeah, more negative margins...not a fan.  If you want to reduce that gap, lower the margin on the element below.

File: styles.css [hunk 25]
```diff
@@ -0,0 +1,391 @@
...
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    display: block !important;
+  }
+
+ /* Mobile Styles */
+ 
+  @media screen and (max-width: 800px) {
```
Comment (jimmynono) [inline styles.css]: We'll come back to this later on in the course., but a great idea to include it now. Some of the homework later on will be a bit easier.

File: styles.css [hunk 26]
```diff
@@ -0,0 +1,391 @@
...
+    height: 2.5em;
+    margin-top: 2rem;
+    text-shadow: none !important;
+    border: none !important;
+    box-shadow:none !important;
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 700;
+    cursor: pointer;
+  }
+
```
Comment (jimmynono) [inline styles.css]: You only want one empty line at the end of files

**Pull request closed without merging**